ice_cube.gemspec

```diff
@@ -22,4 +22,5 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency('rake')
   s.add_development_dependency('rspec', '> 3')
+  s.add_development_dependency('timecop')
```

Comment:
Just curious, what do you use it for? I always just stub `Time.now` and never found a need for this gem! 😉